streams/src/test/java/org/apache/kafka/streams/processor/internals/ActiveTaskCreatorTest.java

```diff
@@ -77,161 +67,11 @@ public class ActiveTaskCreatorTest {
         mkEntry(StreamsConfig.APPLICATION_ID_CONFIG, "appId"),
         mkEntry(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, "dummy:1234")
     );
-    final UUID uuid = UUID.randomUUID();
-
-    private ActiveTaskCreator activeTaskCreator;
-
-    // non-EOS test
-
-    // functional test
-
-    @Test
-    public void shouldConstructProducerMetricsWithEosDisabled() {
```

Comment:
These removed tests seems not recovered in the TaskManagerTest, I wonder if we would lose their testing coverage?

Reply:
Ack.